src/contracts/core/DelegationManager.sol

```diff
@@ -809,117 +811,121 @@ contract DelegationManager is
     /**
      * @notice Returns 'true' if `staker` *is* actively delegated, and 'false' otherwise.
      */
-    function isDelegated(address staker) public view returns (bool) {
+    function isDelegated(
+        address staker
+    ) public view returns (bool) {
         return (delegatedTo[staker] != address(0));
     }
 
     /**
      * @notice Returns true is an operator has previously registered for delegation.
      */
-    function isOperator(address operator) public view returns (bool) {
+    function isOperator(
+        address operator
+    ) public view returns (bool) {
         return operator != address(0) && delegatedTo[operator] == operator;
     }
 
     /**
      * @notice Returns the OperatorDetails struct associated with an `operator`.
      */
-    function operatorDetails(address operator) external view returns (OperatorDetails memory) {
+    function operatorDetails(
+        address operator
+    ) external view returns (OperatorDetails memory) {
         return _operatorDetails[operator];
     }
 
     /**
      * @notice Returns the delegationApprover account for an operator
      */
-    function delegationApprover(address operator) external view returns (address) {
+    function delegationApprover(
+        address operator
+    ) external view returns (address) {
         return _operatorDetails[operator].delegationApprover;
     }
 
     /**
      * @notice Returns the stakerOptOutWindowBlocks for an operator
      */
-    function stakerOptOutWindowBlocks(address operator) external view returns (uint256) {
+    function stakerOptOutWindowBlocks(
+        address operator
+    ) external view returns (uint256) {
         return _operatorDetails[operator].stakerOptOutWindowBlocks;
     }
 
-    /// @notice Given array of strategies, returns array of shares for the operator
-    function getOperatorShares(
-        address operator,
+    /// @notice a legacy function that returns the total delegated shares for an operator and strategy
+    function operatorShares(address operator, IStrategy strategy) public view returns (uint256) {
+        uint64 totalMagnitude = allocationManager.getTotalMagnitude(operator, strategy);
+        return operatorDelegatedShares[operator][strategy].toOwnedShares(totalMagnitude).unwrap();
+    }
+
+    /**
+     * @notice Given a staker and a set of strategies, return the shares they can queue for withdrawal.
+     * This value depends on which operator the staker is delegated to.
+     * The shares amount returned is the actual amount of Strategy shares the staker would receive (subject
+     * to each strategy's underlying shares to token ratio).
+     */
+    function getDelegatableShares(
+        address staker,
         IStrategy[] memory strategies
-    ) public view returns (uint256[] memory) {
-        uint256[] memory shares = new uint256[](strategies.length);
+    ) public view returns (OwnedShares[] memory ownedShares) {
```

Comment:
Weird to have `getDelegatableShares` return `OwnedShares[]`

... rather than `DelegatableShares[]` or something...

Reply:
This is something @gpsanant and I discussed, working on it..